CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 0.16.0 - <date>
+
+### Changed
+- Bump websockets version to 10.0 on Python 3.7+ (#1180)
```

Comment:
```suggestion
- Bump websockets version to 10.0 on Python 3.7+ (#1180)

### Fixed

- Terminate reload process in case startup event fails (#1177)
```